test/plain-list/AutosuggestApp.test.js

```diff
@@ -284,6 +576,38 @@ describe('Plain list Autosuggest', () => {
     });
   });
 
+  describe('getSuggestionValue when selectFirstSuggestion is true', () => {
+    before(() => {
+      setSelectFirstSuggestion(true);
+    });
+
+    after(() => {
+      setSelectFirstSuggestion(false);
+    });
+
+    beforeEach(() => {
+      focusAndSetInputValue('p');
+      getSuggestionValue.reset();
+    });
+
+    it('should not be called when Up is pressed (goes to input field)', () => {
+      clickUp();
+      expect(getSuggestionValue).not.to.have.been.called;
+    });
+
+    it('should be called once with the right parameters when Down is pressed', () => {
+      clickDown();
+      expect(getSuggestionValue).to.have.been.calledOnce;
+      expect(getSuggestionValue).to.have.been.calledWithExactly({ name: 'PHP', year: 1995 });
+    });
+
+    it('should be called once with the right parameters when suggestion is clicked', () => {
+      clickSuggestion(2);
+      expect(getSuggestionValue).to.have.been.calledOnce;
+      expect(getSuggestionValue).to.have.been.calledWithExactly({ name: 'Python', year: 1991 });
+    });
+  });
```

Comment:
I suggest instead of adding this `describe` we add one more test to the `describe` above:

    it('should be called once with the right parameters when input is focused', ...)